webextensions/api/runtime.json

```diff
@@ -753,7 +753,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/runtime/lastError",
             "support": {
               "chrome": {
-                "version_added": true,
+                "version_added": "≤56",
```

Comment:
@queengooborg according to the [version history list](https://google.fandom.com/wiki/Chrome_version_history) I'm using on May 3, 2017 version 58 was the release version.